.rubocop.yml

```diff
@@ -32,3 +32,5 @@ Lint/ShadowedException:
 Layout/AlignHash:
   Enabled: true
   EnforcedLastArgumentHashStyle: ignore_implicit
+Style/NestedTernaryOperator:
+  Enabled: false
```

Comment:
This is a style warning for a good reason, nested ternaries are hard to read and understand. Let's find a better way to do this. Maybe we need a private method that we can call to coerce out input value.

Reply:
Agreed!